src/SprykerSdk/Spryk/Model/Spryk/Filter/SingularizeFilter.php

```diff
@@ -28,6 +28,8 @@ public function getName(): string
      */
     public function filter(string $value): string
     {
-        return Inflector::singularize($value);
+        $inflector = InflectorFactory::create()->build();
+
+        return $inflector->singularize($value);
```

Comment:
I took care of that in my PR https://github.com/spryker-sdk/spryk/pull/61
Maybe you can rebase?

Reply:
We migth also want to look into
```
Class SprykerSdkTest\Spryk\Integration\Glue\AddGlueProcessorModelInterfaceMethodTest located in ./tests/SprykerSdkTest/Spryk/Integration/Glue/Processor/AddGlueProcessorModelInterfaceMethodTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\Glue\Processor\AddGlueResourceRestResponseBuilderModelInterfaceTest located in ./tests/SprykerSdkTest/Spryk/Integration/Glue/Processor/ResponseBuilder/AddGlueResourceRestResponseBuilderModelInterfaceTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\Glue\Processor\AddGlueResourceRestResponseBuilderModelTest located in ./tests/SprykerSdkTest/Spryk/Integration/Glue/Processor/ResponseBuilder/AddGlueResourceRestResponseBuilderModelTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\Glue\AddGlueProcessorModelMethodTest located in ./tests/SprykerSdkTest/Spryk/Integration/Glue/Processor/AddGlueProcessorModelMethodTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\Glue\Processor\AddGlueResourceMapperTest located in ./tests/SprykerSdkTest/Spryk/Integration/Glue/Processor/Mapper/AddGlueResourceMapperTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\Glue\Processor\AddGlueResourceMapperInterfaceTest located in ./tests/SprykerSdkTest/Spryk/Integration/Glue/Processor/Mapper/AddGlueResourceMapperInterfaceTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\Zed\Dependency\Client\AddGlueDependencyClientInterfaceTest located in ./tests/SprykerSdkTest/Spryk/Integration/Glue/Dependency/Client/AddGlueDependencyClientInterfaceTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\Zed\Dependency\Client\AddGlueDependencyClientDependencyProviderConstantTest located in ./tests/SprykerSdkTest/Spryk/Integration/Glue/Dependency/Client/AddGlueDependencyClientDependencyProviderConstantTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\Zed\Dependency\Client\AddGlueDependencyClientFactoryMethodTest located in ./tests/SprykerSdkTest/Spryk/Integration/Glue/Dependency/Client/AddGlueDependencyClientFactoryMethodTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\Zed\Dependency\Facade\AddGlueDependencyClientBridgeTest located in ./tests/SprykerSdkTest/Spryk/Integration/Glue/Dependency/Client/AddGlueDependencyClientBridgeTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\Zed\Dependency\Client\AddGlueDependencyClientDependencyProviderMethodTest located in ./tests/SprykerSdkTest/Spryk/Integration/Glue/Dependency/Client/AddGlueDependencyClientDependencyProviderMethodTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\Zed\Dependency\Client\AddGlueDependencyClientBridgeMethodTest located in ./tests/SprykerSdkTest/Spryk/Integration/Glue/Dependency/Client/AddGlueDependencyClientBridgeMethodTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\Zed\Dependency\Client\AddGlueDependencyClientBridgeInterfaceMethodTest located in ./tests/SprykerSdkTest/Spryk/Integration/Glue/Dependency/Client/AddGlueDependencyClientBridgeInterfaceMethodTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\AddClientConfigMethodTest located in ./tests/SprykerSdkTest/Spryk/Integration/Client/AddClientConfigMethodTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\AddYvesConfigTest located in ./tests/SprykerSdkTest/Spryk/Integration/Yves/AddYvesConfigTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\Yves\Dependency\Facade\AddYvesDependencyClientBridgeTest located in ./tests/SprykerSdkTest/Spryk/Integration/Yves/Dependency/Client/AddYvesDependencyClientBridgeTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\AddYvesConfigMethodTest located in ./tests/SprykerSdkTest/Spryk/Integration/Yves/AddYvesConfigMethodTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\AddSharedRestAttributesTransferTest located in ./tests/SprykerSdkTest/Spryk/Integration/Shared/AddSharedRestAttributesTransferTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\AddSharedConstantsInterfaceTest located in ./tests/SprykerSdkTest/Spryk/Integration/Shared/AddSharedConstantsInterfaceTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\AddSharedTransferSchemaTest located in ./tests/SprykerSdkTest/Spryk/Integration/Shared/AddSharedTransferSchemaTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\Zed\Dependency\Facade\AddZedDependencyClientBridgeTest located in ./tests/SprykerSdkTest/Spryk/Integration/Zed/Dependency/Client/AddZedDependencyClientBridgeTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\AddZedCommunicationTestSuiteTest located in ./tests/SprykerSdkTest/Spryk/Integration/Zed/Communication/AddZedCommunicationTestSuiteTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\AddZedCommunicationTestSuiteConfigurationTest located in ./tests/SprykerSdkTest/Spryk/Integration/Zed/Communication/AddZedCommunicationTestSuiteConfigurationTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\AddZedBusinessFacadeInterfaceTest located in ./tests/SprykerSdkTest/Spryk/Integration/Zed/Business/AddZedBusinessFacadeInterfaceTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\AddZedBusinessFactoryMethodTest located in ./tests/SprykerSdkTest/Spryk/Integration/Zed/Business/AddZedBusinessFactoryMethodTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\AddZedBusinessTestSuiteConfigurationTest located in ./tests/SprykerSdkTest/Spryk/Integration/Zed/Business/AddZedBusinessTestSuiteConfigurationTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\AddZedBusinessFacadeMethodTestTest located in ./tests/SprykerSdkTest/Spryk/Integration/Zed/Business/AddZedBusinessFacadeMethodTestTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\AddZedBusinessTestSuiteTest located in ./tests/SprykerSdkTest/Spryk/Integration/Zed/Business/AddZedBusinessTestSuiteTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\AddZedBusinessFacadeInterfaceMethodTest located in ./tests/SprykerSdkTest/Spryk/Integration/Zed/Business/AddZedBusinessFacadeInterfaceMethodTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\AddZedBusinessFacadeTest located in ./tests/SprykerSdkTest/Spryk/Integration/Zed/Business/AddZedBusinessFacadeTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\AddZedBusinessFacadeMethodTest located in ./tests/SprykerSdkTest/Spryk/Integration/Zed/Business/AddZedBusinessFacadeMethodTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\AddZedBusinessModelTest located in ./tests/SprykerSdkTest/Spryk/Integration/Zed/Business/Model/AddZedBusinessModelTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\AddZedBusinessModelMethodTest located in ./tests/SprykerSdkTest/Spryk/Integration/Zed/Business/Model/AddZedBusinessModelMethodTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\AddZedBusinessModelInterfaceTest located in ./tests/SprykerSdkTest/Spryk/Integration/Zed/Business/Model/AddZedBusinessModelInterfaceTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\AddZedBusinessModelInterfaceMethodTest located in ./tests/SprykerSdkTest/Spryk/Integration/Zed/Business/Model/AddZedBusinessModelInterfaceMethodTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\AddZedBusinessFactoryTest located in ./tests/SprykerSdkTest/Spryk/Integration/Zed/Business/AddZedBusinessFactoryTest.php does not comply with psr-4 autoloading standard. Skipping.
Class SprykerSdkTest\Spryk\Integration\AddZedBusinessFacadeTestTest located in ./tests/SprykerSdkTest/Spryk/Integration/Zed/Business/AddZedBusinessFacadeTestTest.php does not comply with psr-4 autoloading standard. Skipping.
```

Reply:
Oh, and some of the tests make no sense.

There are assert.. after expectException() call
Any assignment or test afterwards is a silent no op and should be removed
E.g.

```
        $this->expectException(SprykWrongDevelopmentLayerException::class);

        $this->tester->run($this, [
            '--module' => 'FooBar',
            '--mode' => 'project',
        ]);

        $this->assertFileExists($this->tester->getModuleDirectory() . 'codeception.yml'); // Noise
```

Reply:
I opened a ticket for a sniff for core as well: https://github.com/spryker/code-sniffer/issues/252
We should disallow these code smells in all our repos in the future.

Reply:
Fixed all the stuff.